examples/ads.amp.html

```diff
@@ -181,6 +180,7 @@
         <option>pubmine</option>
         <option>pulsepoint</option>
         <option>purch</option>
+        <option>capirs</option>
```

Comment:
oops please revert the change here. Names are sorted by alphabetical order.

Reply:
done